migrate/migrations/20191216145826-calendar-subscriptions.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+
+CREATE TABLE user_calendar_subscriptions (
+    id UUID PRIMARY KEY,
+    name TEXT NOT NULL,
+    user_id UUID NOT NULL REFERENCES users (id) ON DELETE CASCADE,
+    last_access TIMESTAMPTZ NOT NULL DEFAULT now(),
```

Comment:
As per the UI discussion, I think this will be nullable (and default to null) and we're going to add a `last_updated` field that is NOT NULL defaulting to `now()` and set when updated.